packages/ember/tailwind-vite/index.js

```diff
@@ -0,0 +1,92 @@
+// @ts-check
+import { css, js, packageJson } from "ember-apply";
+// eslint-disable-next-line n/no-unpublished-import
+import { execa } from "execa";
+// eslint-disable-next-line n/no-unpublished-import
+import fse from "fs-extra";
+import { dirname } from "path";
+import { fileURLToPath } from "url";
+
+// @ts-ignore
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+export default async function run() {
+  await packageJson.addDevDependencies({
+    tailwindcss: "^3.0.0",
+    postcss: "^8.0.0",
+    autoprefixer: "^10.0.0",
+  });
+
+  await execa("npx", ["tailwindcss", "init", "-p"]);
+
+  await css.transform("app/styles/app.css", {
```

Comment:
I don't think we can use the styles directory. It's corrupted with broccoli logic, yea?

For now, we can colocate the css file with the app.js/ts